docs/src/main/sphinx/connector/clickhouse.rst

```diff
@@ -46,6 +46,8 @@ catalog for each server. To add another catalog:
 For example, if you name the property file ``sales.properties``, Trino uses the
 configured connector to create a catalog named ``sales``.
 
+.. include:: jdbc-common-configurations.fragment
+
```

Comment:
Would suggest moving this section above "Multiple ClickHouse servers", or around line 36. Configuration properties are more generally applicable than the above section.